packages/core/state/interaction/reducer.ts

```diff
@@ -90,7 +85,6 @@ export const initialState: InteractionStateBranch = {
         fileViewerService: new FileViewerServiceNoop(),
         frontendInsights: new FrontendInsights({
             application: {
-                // Kept old name to compare usage more easily in Amplitude UI
```

Comment:
Possibly still a relevant comment?

Reply:
resolved in [07a2715](https://github.com/AllenInstitute/biofile-finder/pull/339/commits/07a27155752327761b2e69939c1a7a4e09154fa3)